crawler.py

```diff
@@ -186,21 +187,48 @@ def __init__(self, args):
 
         self.storage_objects = ['snitch_map', 'action_map']
 
+        # create an XVFB virtual display (to avoid opening an actual browser)
+        self.vdisplay = Xvfb(width=1280, height=720)
+        self.vdisplay.start()
+        self.start_browser()
+
+        browser_version = self.driver.capabilities["browserVersion"]
+
+        # returns most recent commit hash
+        def bundle_git_info(path):
+            git_info = []
```

Comment:
Nit: let's use a dict so that it's clear what's what. `git_data[0]` vs. `git_data['branch']`.